cmd/commands/commands.go

```diff
@@ -2267,10 +2267,24 @@ func listChainTxns(ctx *cli.Context) error {
 		MaxTransactions: uint32(ctx.Uint64("max_transactions")),
 	}
 
-	if ctx.IsSet("start_height") {
+	switch {
+	case ctx.IsSet("start_height") && ctx.IsSet("end_height"):
+		startHeight := ctx.Int64("start_height")
+		endHeight := ctx.Int64("end_height")
+
+		if endHeight != -1 && startHeight > endHeight {
+			return errors.New("start_height should " +
+				"be less than end_height if end_height" +
```

Comment:
nit: always put space between words on previous line for consistency. If the line becomes too long, break a word earlier.